ingestion-sink/src/test/resources/testdata/payload-format-expected.ndjson

```diff
@@ -13,8 +13,9 @@
 {"additional_properties":{"missingField":null}}
 {"additional_properties":{"test_map_without_value":{"key":"value"}},"test_map":[{"key":"key","value":"value"}],"test_map_without_value":[{"key":"key"}]}
 {"additional_properties":{},"test_repeated_record":[{"field":"value"}],"test_record":{"key":"value"}}
-{"additional_properties":{"test_repeated_record":[null,{"reject":"value","alsoReject":null},null]},"test_repeated_record":[{"field":"value"},{},{}]}
-{"additional_properties":{"test_repeated_record":[{"reject":"value"},null,{"reject":"value"}]},"test_repeated_record":[{},{"field":"value"},{}]}
+{"additional_properties":{"test_repeated_record":[{},{"reject":"value","alsoReject":null},{}]},"test_repeated_record":[{"field":"value"},{},{}]}
+{"additional_properties":{"test_repeated_record":[{"reject":"value"},{},{"reject":"value"}]},"test_repeated_record":[{},{"field":"value"},{}]}
+{"additional_properties":{"test_repeated_record":[null,{}]},"test_repeated_record":[{"field":"value"}]}
```

Comment:
I don't think we should ever remove an entry from a list. Instead I think we should insert an empty object where possible, maybe an empty array if all of the values are empty objects, and otherwise send the whole list to `additional_properties`.
```suggestion
{"additional_properties":{"test_repeated_record":[null,{}]},"test_repeated_record":[{},{"field":"value"}]}
```